spec/dummy/config/environments/test.rb

```diff
@@ -0,0 +1,10 @@
+Rails.application.configure do
+  config.active_storage.service = :test
+  config.action_mailbox.ingress = :amazon
+  config.action_mailbox.amazon.subscribed_topics = %w(
+    arn:aws:sns:eu-west-1:111111111111:example-topic
+    arn:aws:sns:eu-west-1:111111111111:recognized-topic
```

Comment:
You don't have to do this, but we typically use 012345678910 as account ID because it's easier to confirm typos of 12 digits.

Reply:
(this relates to the fixtures, so it's tangled)

Reply:
TBD